src/controllers/connection/newConnection.ts

```diff
@@ -218,16 +221,17 @@ export class NewConnectionController extends HTMLController {
   @SuccessResponse(200)
   @Post('/pin-submission')
   public async submitPinCode(
-    @Body() body: { action: 'submitPinCode'; pin: string; companyNumber: COMPANY_NUMBER }
+    @Body() body: { action: 'submitPinCode'; pin: string; connectionId: UUID }
```

Comment:
connection_id should be in the path not the body. Also can we please move these two methods to the `connection` controller (`index.ts`) so that the paths are `/v1/connection/{uuid}/pin-submission`